<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmCaptureMode</class>
 <widget class="QWidget" name="CArmCaptureMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CArmCaptureMode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:rgba(14,14,14,1);
}
QLabel{
background:transparent;
font-size:14px;
font-family:NotoSansCJK-DemiLight,NotoSansCJK;
font-weight:normal;
color:rgba(180,196,217,1);
line-height:21px;
}
</string>
  </property>
  <widget class="QPushButton" name="pulsePerspectiveBtn">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:rgba(0,0,0,0);
opacity:0.7;
image:url(:/modelParts/Resource/modelParts/pulse_perspective.png);
}
QPushButton:checked{
image:url(:/modelParts/Resource/modelParts/pulse_perspective_checked.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cineBtn">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:rgba(0,0,0,0);
opacity:0.7;
image:url(:/modelParts/Resource/modelParts/cine.png);
}
QPushButton:checked{
image:url(:/modelParts/Resource/modelParts/cine_checked.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>60</y>
     <width>28</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>60</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>脉冲透视</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pointCaptureBtn">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
image:url(:/modelParts/Resource/modelParts/point_caputre.png);
background:rgba(0,0,0,0);
opacity:0.7;
}

QPushButton:checked{
image:url(:/modelParts/Resource/modelParts/point_capture_cheched.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>60</y>
     <width>28</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>点拍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="togglePartBtn">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>38</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:rgba(0,0,0,0);
opacity:0.7;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pulseLbe">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>80</y>
     <width>222</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background:rgba(0,0,0,1);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="partLbe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>105</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
font-size:14px;
font-family:NotoSansCJK-Regular,NotoSansCJK;
font-weight:400;
color:rgba(180,196,217,1);
line-height:21px;
text-align:right;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="togglePart_2Btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>112</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
font-size:14px;
font-family:NotoSansCJK-Regular,NotoSansCJK;
font-weight:400;
color:rgba(180,196,217,1);
line-height:21px;
text-align:left;
image:url(:/modelParts/Resource/modelParts/triangle.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CArmTouchPanel.qrc"/>
 </resources>
 <connections/>
</ui>
